### PR TITLE
Add support of Jinja2 "Do" tag

### PR DESCRIPTION
**Pull request by n-rodriguez**
From Jinja2 doc : https://jinja.palletsprojects.com/en/2.11.x/extensions/#expression-statement

Thank you!

### CODE REVIEW

File: spec/tags/do_spec.cr [hunk 1]
```diff
@@ -0,0 +1,37 @@
+require "../spec_helper.cr"
+
+struct Crinja::Value
+  def push(value)
```
Comment (straight-shoota) [inline spec/tags/do_spec.cr:4]: I'd rather not have this method in the public namespace, even if it's just in the specs.
It's also unnecessary: You can just call `array.as_a.push(item)` in the function implementation.

**Comment by n-rodriguez** [comment]
> Could you add some documentation to the tag class? A description and link to jinja2 docs (i.e. the PR message) plus a short usage example would be great.

Done!

> I'm not sure about adding it to default tags. So far, most of Crinja's library should be mostly compatible with Jinja2 defaults. So maybe it would be better to have this tag as an extension in Crinja, too.

At the beginning I wanted to implement the `do` on my side using  `env.tags << MyTag.new` but it didn't work so I ended up with this PR.

**Comment by straight-shoota** [comment]
> At the beginning I wanted to implement the do on my side using env.tags << MyTag.new but it didn't work so I ended up with this PR.

Oh, that should really work. Can you still reproduce that? It's definitely an error.
I mean the code in `lib/tag.cr` doesn't do anything else.

Nonetheless, it surely makes sense to have the `do` tag in Crinja's default library. So thanks for this PR!
I'm just not 100% confident whether it should be enabled by default (i.e. included in `TAGS` list used in `register_defaults`).

**Comment by n-rodriguez** [comment]
> Oh, that should really work. Can you still reproduce that? It's definitely an error.

Hi there! Finally I could make it work in my own project. Do you still want this PR? I think it could be useful for other people too.

File: src/lib/tag/do.cr [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# The `do` tag works exactly like the regular variable expression ({{ ... }}) except it doesn’t print anything.
```
Comment (straight-shoota) [inline src/lib/tag/do.cr:1]:
```suggestion
# The `do` tag works exactly like the regular variable expression (`{{ ... }}`) except it doesn’t print anything.
```

**Comment by n-rodriguez** [comment]
>  I'd definitely like to add this, and a apart from a minor doc improvement this looks good to merge!

done!

**Comment by straight-shoota** [comment]
Awesome, thanks!

**Pull request merged**